src/porepy/numerics/fem/rt0.py

```diff
@@ -74,9 +74,10 @@ def discretize(self, g, data):
         # Map the domain to a reference geometry (i.e. equivalent to compute
         # surface coordinates in 1d and 2d)
         deviation_from_plane_tol = data.get("deviation_from_plane_tol", 1e-5)
```

Comment:
As a general comment, I think this is a rather loose tolerance for deviations; in practice we need to be much stricter not to get problems in the meshing. But that probably is an argument that the test is not much needed.

Another issues is that it is stupid not to have a relative tolerance here, but that is common to all computational geometry in pp, and it is my bad.

Reply:
yes, I agree with you..